packages/release/prepare-local-fs.service

```diff
@@ -10,8 +10,9 @@ RefuseManualStop=true
 [Service]
 Type=oneshot
 
+EnvironmentFile=/usr/lib/data-partition-filesystem
```

Comment:
Instead of an environment variable, what if `prepare-local-fs.service` becomes an `in` file, and in the `ExecStart` directive you use `DATA_PARTITION_FILESYSTEM` to replace it with `sed`?